src/game.py

```diff
@@ -3,21 +3,58 @@
 from src.rules import Rules
 from src.user_interface import UserInterface
 from src.validator import Validator
+from src.symbol import SymbolOptions
 
 
 class Game:
     def __init__(self, ui=UserInterface(), message=Message()):
         self.board = Board()
         self.rules = Rules()
         self.validator = Validator()
+        self.symbol = SymbolOptions()
         self.ui = ui
         self.message = message
         self.game_board = self.board.starter_board
-        self.player_one = self.board.player_one
-        self.player_two = self.board.player_two
+        self.player_one = "X"
+        self.player_two = "O"
         self.total_marks_on_board = 0
         self.playing = True
 
+    def change_symbols(self):
+        self.ui.display_message(self.message.menu())
+        user_input = self.ui.get_user_input()
+        valid_user_input = self.validator.is_valid_menu_choice(user_input)
+        while valid_user_input is False:
+            self.ui.display_message(self.message.invalid_choose_symbol_input())
+            user_input = self.ui.get_user_input()
+            valid_user_input = self.validator.is_valid_menu_choice(user_input)
+        if user_input == "2":
+            self.ui.display_message(self.message.display_symbols())
+            self.set_player_one_symbol()
+            self.set_player_two_symbol()
+
+    def set_player_one_symbol(self):
+        self.ui.display_message(self.message.choose_symbol_player_one())
+        symbol = self.ui.get_user_input()
+        valid_symbol = self.validator.is_valid_symbol_choice_input(symbol)
+        while not valid_symbol:
+            self.ui.display_message(self.message.invalid_symbol_option())
+            symbol = self.ui.get_user_input()
+            valid_symbol = self.validator.is_valid_symbol_choice_input(symbol)
+        self.player_one = self.symbol.get_symbols(symbol)
+        return self.player_one
+
+    def set_player_two_symbol(self):
+        self.ui.display_message(self.message.choose_symbol_player_two())
+        symbol = self.ui.get_user_input()
+        valid_symbol = self.validator.is_valid_symbol_choice_input(symbol)
+        while not valid_symbol:
+            self.ui.display_message(self.message.invalid_symbol_option())
+            symbol = self.ui.get_user_input()
+            valid_symbol = self.validator.is_valid_symbol_choice_input(symbol)
+        self.player_two = self.symbol.get_symbols(symbol)
+        return self.player_two
+
```

Comment:
can we make this more generic by using parameters?

Reply:
Working on this now and cannot for the life of me get it to work...